basic/stam/StamExAnswer.py

```diff
@@ -0,0 +1,35 @@
+"""Reads a file, edits it and saves it in a new location."""
+import sys
+
+INPUT_FILE = 1
+OUTPUT_FILE = 2
+FIRST_NAME = "Keren"
+LAST_NAME = "Guez"
+
+
+def stam(filename, output_file):
+    """Get a file, edit it and save it to a new location.
+
+    Args:
+        filename (str): The path to the file that will be edited.
+        output_file (str) : The path of the new file to be saved.
+    """
+    with open(filename, 'r') as file:
+        with open(output_file, 'w') as new_file:
+            for line in file:
+                new_line = f"{FIRST_NAME}{line[:-1]}{LAST_NAME}\n"
+                new_file.write(new_line)
+
+
+def main():
+    """Initialize input and output paths and call stam function."""
+    try:
+        path = sys.argv[INPUT_FILE]
+        new_path_to_save = sys.argv[OUTPUT_FILE]
+        stam(path, new_path_to_save)
+    except Exception as e:
+        print(e)
```

Comment:
The try except here are pointless.
At the moment, you catch the exception and just print it as it is and ending the program. If this the case, you can just remove the try except, and let it throw exception if it fails.
If you do want to use try except here, you should do something different, such as printing your own error message